### DISCUSSION TITLE
error fuel nozzle

### DISCUSSION THREAD

**Discussion by Prometeo-ianic** [firemodels/fds#13196] [General]
Hi,
I tried to create a spray of C14H30 by modifying the validation case 'US Coast Guard Water Mist Tests'. Unfortunately I get the error:

"ERROR: PARTicle class C14H30 requires a TMP_V (CHID: B2)

 DEVICE Activation Times

   1  fuel_nozzle                                                 No Activation

STOP: FDS was improperly set-up (CHID: B2)
STOP: FDS was improperly set-up (CHID: B2)"

This is the text.  Could you please help me to detect which error I made?
Thanks

&HEAD CHID='B2', TITLE='Case1'/
&TIME T_END=600.0/
&MESH ID='Mesh01', IJK=70,50,30, XB=0.0,7.0,0.0,5.0,0.0,3.0/

&SPEC ID='C14H30', FORMULA='C14H30'/

&PART ID='C14H30',
      SPEC_ID='C14H30',
      DIAMETER=200.0/

&REAC ID='C14H30',
      FYI='C14H30',
      FUEL='C14H30',
      AUTO_IGNITION_TEMPERATURE=0.0,
      CO_YIELD=6.0E-3,
      SOOT_YIELD=0.015/

&PROP ID='Bete P54',
      PART_ID='C14H30',
      FLOW_RATE=2.19,
      PARTICLE_VELOCITY=8.0,
      SPRAY_ANGLE=0.0,15.0/
&DEVC ID='fuel_nozzle', PROP_ID='Bete P54', XYZ=3.5,1.8,1.0, ORIENTATION=0.0,0.0,1.0, QUANTITY='TIME', SETPOINT=0.0/

&TAIL /

**Comment by rmcdermo** [comment]
Per user guide section on liquid droplets:
```
Unknown Liquid/Gas Species
If the SPEC_ID is not listed in Appendix A with a “Y” in the “Liquid” column, you must provide all of the following properties of the liquid on the SPEC line:
DENSITY_LIQUID (kg/m3).
SPECIFIC_HEAT_LIQUID (kJ/(kgK)). If the specific heat of the liquid is a function of temperature, use
RAMP_CP_L to provide the function cp(T).
VAPORIZATION_TEMPERATURE Boiling temperature of the liquid, Tboil (◦C).
MELTING_TEMPERATURE Melting (solidification) temperature of the liquid (◦C). ENTHALPY_OF_FORMATION The heat of formation of the gas (kJ/mol).
HEAT_OF_VAPORIZATION Latent heat of vaporization of the liquid, hv (kJ/kg). H_V_REFERENCE_TEMPERATURE The temperature corresponding to the HEAT_OF_VAPORIZATION (◦C). VISCOSITY_LIQUID The viscosity of the liquid (kg/(m s)).
CONDUCTIVTY_LIQUID The conductivity of the liquid (W/(m K)).
BETA_LIQUID The coefficient of thermal expansion of the liquid (1/K).
```

**Comment by drjfloyd** [comment]
You error is exactly as the message states. You did not specify a TMP_V. Search the User's Guide for TMP_V.

**Reply by drjfloyd** [reply]
You appear to be running an old version of FDS. The current error message is:

'ERROR(182): SPEC ',TRIM(SS%ID),' does not have a VAPORIZATION_TEMPERATURE.'

This change was made about two years ago.

**Reply by Prometeo-ianic** [reply]
Oh, yes. You are right, I am using my old pc, (my new one is at the reparation store). I install the new one also here.  Thank you.

**Comment by Prometeo-ianic** [comment]
Hi,
I am trying to fix my model, but I only see the particle without a fire. 
I get the warning that the SPEC are not assigned but it seems assigned to me.
Thank you in advace.

&HEAD CHID='B2', TITLE='Case1'/
&TIME T_END=600.0/
&MESH ID='Mesh01', IJK=40,40,40, XB=0.0,5.0,0.0,5.0,0.0,5.0/

&SPEC ID='C40H82O4', FORMULA='C40H82O4', SPECIFIC_HEAT=2.0, DENSITY_LIQUID=850.0, SPECIFIC_HEAT_LIQUID=2.0, VAPORIZATION_TEMPERATURE=350.0, MELTING_TEMPERATURE=-50.0, HEAT_OF_VAPORIZATION=250.0, ENTHALPY_OF_FORMATION=-500.0, H_V_REFERENCE_TEMPERATURE=25.0/

&PART ID='C40H82O4',
      SPEC_ID='C40H82O4',
      DIAMETER=200.0,
      RGB=255,255,51/

&REAC ID='C40H82O4',
      FYI='C14H30',
      FUEL='C40H82O4',
      CO_YIELD=0.02,
      SOOT_YIELD=0.03,
      HEAT_OF_COMBUSTION=45000.0/

&PROP ID='Bete P54',
      PART_ID='C40H82O4',
      FLOW_RATE=0.76,
      PARTICLE_VELOCITY=16.18,
      SPRAY_ANGLE=0.0,15.0/
&DEVC ID='fuel_nozzle', PROP_ID='Bete P54', XYZ=3.5,1.8,1.0, ORIENTATION=0.0,0.0,1.0, QUANTITY='TIME', SETPOINT=0.0/

&VENT ID='Mesh Vent: Mesh01 [XMAX]', SURF_ID='OPEN', XB=5.0,5.0,0.0,5.0,0.0,5.0/ 
&VENT ID='Mesh Vent: Mesh01 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.0,5.0,0.0,5.0/ 
&VENT ID='Mesh Vent: Mesh01 [YMAX]', SURF_ID='OPEN', XB=0.0,5.0,5.0,5.0,0.0,5.0/ 
&VENT ID='Mesh Vent: Mesh01 [YMIN]', SURF_ID='OPEN', XB=0.0,5.0,0.0,0.0,0.0,5.0/ 
&VENT ID='Mesh Vent: Mesh01 [ZMAX]', SURF_ID='OPEN', XB=0.0,5.0,0.0,5.0,5.0,5.0/ 
&TAIL /

**Comment by mcgratta** [comment]
This warning means that FDS does not have properties for your gas species. It can only assume its molecular weight from the formula. All other properties are assumed to be that of nitrogen.

**Reply by drjfloyd** [reply]
A little more detail on Kevin's statement.

FDS potentially needs all the gas properites discussed in the species chapter of the guide: MW, specific heat, Gibbs energy, heat of formation, viscosity, and diffusivity.

The key property of a fuel being used for LES with simple chemistry is specific heat. The assumption of gamma=1.4 is not a good assumption for high MW species. You have specified this on your SPEC line. 

FDS also needs diffusivity and viscosity, and you have no inputs for those. There it is using the N2 Lennard-Jones parameters. Take a look at the value FDS is getting in the .out file. If they don't look crazy, then you should be OK. Other than a fuel cells close to a burning surface, the fuel species is generally a small mass fraction. In LES the transport uses the turbulent values of those parameters, and the molecular values are used when FDS needs an Re, Sh, or Pr number. 

With simple chemistry Gibb's energy isn't used.

With simple chemistry, since all the products are known species, the FUEL heat of formation can be determined using the HEAT_OF_COMBUSTION or EPUMO2.

**Comment by mcgratta** [comment]
You do not need to add any additional information. FDS has all the
information it needs for the Simple Chemistry combustion model. Just be
aware that FDS is not going to assume anything else about this gas species.
